### PR TITLE
Issue #2978: Add setting to allow choosing which Views display(s) get created by default.

### LINKED ISSUE #2978: [UX] On the Add view page, expand block, and collapse page display
## Describe your issue or idea

Now that we have layouts, I find that I rarely create page displays of views. I create layouts, and put blocks on them. 

If this is the case for most others as well, I'd like to rework the `Add view` page to more prominently feature the block display settings, and less prominently feature the Page display, by switching the order (block first, page second) and by expanding the `Create a block` section, and collapsing the `Create a page` section.

### PR DESCRIPTION
**Pull request by ghost**
Fixes https://github.com/backdrop/backdrop-issues/issues/2978

### CODE REVIEW

**Comment by jenlampton** [comment]
CLosing for new sandbox.

**Comment by jenlampton** [comment]
new sandbox please

File: core/modules/views_ui/views_ui.admin.inc [hunk 1]
```diff
@@ -4494,39 +4494,50 @@ function views_ui_admin_settings_basic() {
...
-  // will make it easier to change if we do.
-  $form['basic'] = array();
+  $form['wizard'] = array(
+    '#type' => 'fieldset',
+    '#title' => t("'Add view' wizard"),
+  );
+  $form['wizard']['wizard_default_display'] = array(
+    '#type' => 'checkboxes',
+    '#title' => t('Default displays'),
+    '#description' => t('Which display options to enable by default. Note that these options can still be manually enabled/disabled before creating the view.'),
```
Comment (bugfolder) [inline core/modules/views_ui/views_ui.admin.inc]:
```suggestion
    '#description' => t('Which displays are enabled by default in the 'Add view' dialog.'),
```
Per https://github.com/backdrop/backdrop-issues/issues/2978#issuecomment-1055242297.

**Comment by quicksketch** [comment]
Merged into 1.x for 1.22.0. Thanks @BWPanda, @bugfolder, and @jenlampton for carrying this to the finish!

**Pull request merged**